src/features/vote/useTableValues.ts

```diff
@@ -125,7 +150,31 @@ export default function useTableValues(
           postData.push(prd);
         }
       });
-      setTableValues(tv);
+
+      const descriptionsAdded = Promise.all(
+        tv.map(async (el) => {
+          const isUmip = el.identifier.includes("Admin");
+          const umipNumber = isUmip
+            ? parseInt(el.identifier.split(" ")[1])
+            : undefined;
+
+          let description = "Price request.";
+
+          if (umipNumber) {
+            description = await (await fetchUmip(umipNumber)).description;
```

Comment:
same thing here, a single call could fail which invalidates all successful calls. you could also do something like [Promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled), which slightly changes what is returned, but at least you can get the results back and filter out errors.

Reply:
I'll look into it, I think the last time I tried that it was a headache with TS.